tests/functional/test_freeze.py

```diff
@@ -818,9 +819,38 @@ def test_freeze_path_multiple(tmpdir, script, data):
     _check_output(result.stdout, expected)
 
 
-def test_freeze_direct_url_archive(script, shared_data, with_wheel):
-    req = "simple @ " + path_to_url(shared_data.packages / "simple-2.0.tar.gz")
-    assert req.startswith("simple @ file://")
-    script.pip("install", req)
-    result = script.pip("freeze")
-    assert req in result.stdout
```

Comment:
Why removing this test?

Reply:
Aah good catch, I removed it by mistake. Readding it